packages/cli/test/unit/commands/git/index.test.ts

```diff
@@ -21,9 +27,22 @@ describe.todo('git', () => {
     });
   });
 
-  it('errors when invoked without subcommand', async () => {
-    client.setArgv('git');
-    const exitCodePromise = git(client);
-    await expect(exitCodePromise).resolves.toBe(2);
+  it('displays help when invoked without subcommand', async () => {
+    const cwd = fixture('new-connection');
+    client.cwd = cwd;
+    try {
+      await fs.rename(join(cwd, 'git'), join(cwd, '.git'));
+      useTeams('team_dummy');
+      useProject({
+        ...defaultProject,
+        id: 'new-connection',
+        name: 'new-connection',
+      });
+      client.setArgv('git');
+      const exitCodePromise = git(client);
+      await expect(exitCodePromise).resolves.toBe(2);
```

Comment:
We should not use the `.resolves` assertion. See: https://github.com/vercel/vercel/pull/12441